packages/ripple-nuxt-tide/lib/templates/request-id.js

```diff
@@ -0,0 +1,19 @@
+import { generateId } from '@dpc-sdp/ripple-nuxt-tide/lib/core/tide-helper'
+import middleware from './middleware'
+
+middleware['request-id'] = async (context) => {
```

Comment:
Added a route middleware for generate request id if it's client side navigation.